addons/account_sequence/models/sequence_mixin.py

```diff
@@ -35,6 +35,7 @@ def _set_next_sequence(self):
                 with self.env.cr.savepoint(flush=False), mute_logger('odoo.sql_db'):
                     self[self._sequence_field] = sequence
                     self.flush_recordset([self._sequence_field])
+                    self.modified([self._sequence_field])
```

Comment:
I don't understand the fix.  The assignment of `self[self._sequence_field]` should automatically invoke `self.modified()` with the given field.  Why isn't it done?  Why is it dependent on modules _account_3way_match_ and _sale_timesheet_enterprise_?  Have you found the root cause of the issue?